src/polls/PollForm.vue

```diff
@@ -199,9 +292,50 @@ export default defineComponent({
     ChangeDate(value: Date) {
       this.pollData.creationDate = value;
     },
+    onChangeCreationDate() {
+      const dateTomorrow = new Date();
+      dateTomorrow.setDate(dateTomorrow.getDate() + 1);
+      dateTomorrow.setHours(0, 0, 0, 0);
+      if (this.pollData.creationDateInEdition < dateTomorrow) {
+        this.isDisabled = true;
+        this.isCreationDateHelpActive = true;
+      } else {
+        this.beginDateInEdition.setHours(0, 0, 0, 0);
+
+        this.finishDate = new Date(this.pollData.creationDateInEdition.getTime() + 14 * 86400000);
+        this.finishDate.setHours(23, 59, 59, 59);
+        this.pollData.completionDate = this.finishDate.toLocaleString('uk-UA').split(',')[0];
+
+        this.isCreationDateHelpActive = false;
+        this.isDisabled = false;
+      }
+    },
+    async initData() {
+      await this.$store.dispatch(
+        `${StoreModuleEnum.pollsStore}/${PollsActionEnum.SET_SELECTED_POLL}`,
+        this.$props.poll.id
+      );
+
+      this.pollData.title = this.selectedPoll?.header ?? '';
+      this.pollData.description = this.selectedPoll?.description ?? 'Повний опис опитування';
+      this.pollData.polledHouses = this.selectedPoll?.polledHouses ?? [];
+      this.pollData.creationDateInEdition = this.selectedPoll?.creationDate.toLocaleString('uk-UA');
+      this.pollData.completionDate = this.selectedPoll?.completionDate.toLocaleString('uk-UA');
+
+      this.beginDateInEdition = this.selectedPoll?.creationDate || new Date();
+      this.finishDate = this.selectedPoll?.completionDate;
+    },
+    submitPollForm() {
+      if (this.isEditing) {
+        this.editPoll();
+      } else {
+        this.createPoll();
+      }
+    },
     async createPoll() {
       const isFormValid = await this.v$.$validate();
       if (!isFormValid) {
+        console.log('i am in isFormValid');
```

Comment:
Do you need this console.log?

Reply:
Absolutely not (missed this console.log before making PR)